migtests/scripts/functions.sh

```diff
@@ -432,6 +442,12 @@ import_data_to_source_replica() {
 	then
 		args="${args} --source-replica-db-schema ${SOURCE_REPLICA_DB_SCHEMA}"
 	fi
+	if [ "${SOURCE_DB_ORACLE_CDB_TNS_ALIAS}" != "" ]
+	then
+		args="${args} --oracle-tns-alias ORCLPDB1SSL"
```

Comment:
You should be taking the actual env var here instead of hard-coding

Reply:
Same as above. Since `SOURCE_DB_ORACLE_TNS_ALIAS` is only getting set in case of offline migration, had to hard-code it here. Can try change this behaviour by adding this variable as well to live_env.sh but then the export-data command will need to have more conditions.

Reply:
https://github.com/yugabyte/yb-voyager/pull/1523/files#r1628896181